packages/expo-audio/src/resolveAssetSource.ts

```diff
@@ -0,0 +1,2 @@
+import resolveAssetSource from 'react-native/Libraries/Image/resolveAssetSource';
```

Comment:
Are you sure this works? So far in the other modules on Web we had to use `@react-native/assets-registry/registry` directly – `react-native-web` itself didn't have `resolveAssetSource` file.